users/commands.md

```diff
@@ -0,0 +1,117 @@
+# Commands
+> When in doubt, check the docs 📄
+
+This section provides documentation about the commands. These commands are updated every time Karen Bot starts, so they are always ✨ fresh ✨
+
+## Regular commands
+These are the commands that get executed by running "m!command"
+<div id="commandsDiv">
+	<table class="sortable">
+		<tr>
+			<th>Type</th>
+			<th>Name</th>
+			<th>Description</th>
+		</tr>
+		<tr v-for="command in commandsObj">
```

Comment:
This whole method is quite ugly, but I suppose it'll suffice for now :/
Add "improving this" to a trello or something